### PR TITLE
add how to share container or group specification

### PR DESCRIPTION
**Pull request by hitohata**
## Overview of Implementation

Added specification documents

### CODE REVIEW

File: project-document/docs/specifications/terms.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+A user in this system is required to register using the Google Account.
+
+* When a user has registered, a new Group and Container for the user will be also created which names are *{user's name}'s group* and *{user's name}'s container* respectively. 
+
+## Container
+
+The Container is a box of foods like a fridge.
+The container must belong to the Group.
+A user who doesn't belong any group **can** create a new container without group, but then the group is created for it automatically.
```
Comment (taniguchiiqqq) [inline project-document/docs/specifications/terms.md:15]: @hitohata 
Now, a user cannot create a new container directly without any group; a user has to choose a group before creating a group.

**Review by taniguchiiqqq — CHANGES_REQUESTED** [review]
Thank you so much. LGTM except for a comment.

File: project-document/docs/specifications/terms.md (hunk 1 above)
Comment (hitohata) [reply project-document/docs/specifications/terms.md]: @taniguchiiqqq 
Thank you. I've fixed it. (deleted it sentence.)

**Review by taniguchiiqqq — APPROVED** [review]
Thanks! LGTM

**Pull request merged**